### ISSUE TITLE
Paper Revision: {2021.eval4nlp-1.*}

### ISSUE THREAD

**Issue by akfujita** [acl-org/acl-anthology#3867]
### Anthology ID

2021.eval4nlp-1.*

### Type of Change

Erratum

### PDF of the Revision or Erratum

Watermarking of all 24 papers in this volume is incorrect: Copyright refers to ACL 2020 and DOI so does RANLP 2019.
e.g., https://doi.org/10.26615/978-954-452-056-4_001 is redirected to https://acl-bg.org/proceedings/2019/RANLP%202019/pdf/RANLP001.pdf
This is neither a metadata issue of individual paper nor a erratum of the paper content, but a publication issue.
Sorry if this form is not an appropriate place to post this request.

### Brief Description of Changes

Watermarking of all 24 papers in this volume is incorrect: Copyright refers to ACL 2020 and DOI so does RANLP 2019.
e.g., https://doi.org/10.26615/978-954-452-056-4_001 is redirected to https://acl-bg.org/proceedings/2019/RANLP%202019/pdf/RANLP001.pdf
This is neither a metadata issue of individual paper nor a erratum of the paper content, but a publication issue.
Sorry if this form is not an appropriate place to post this request.

**Comment by anthology-assist** [comment]
@akfujita We need all the updated PDFs to do a reingestion.

**Comment by akfujita** [comment]
I am not responsible for watermarking.
Then, should I ask the workshop organizers to regenerate PDFs and ask you a reingestion?

**Comment by mbollmann** [comment]
@akfujita Yes please, because the Anthology does not do the watermarking (or any changing of PDF content) either.
